modules/internal/DefaultRectangular.chpl

```diff
@@ -1685,7 +1685,7 @@ module DefaultRectangular {
     // compilation does not work right now.  The calls to chpl_comm_get
     // and chpl_comm_put should be changed once that is fixed.
     if Adata.locale.id==here.id {
-      if debugDefaultDistBulkTransfer then //See bug in test/optimizations/bulkcomm/alberto/rafatest2.chpl
+      if debugDefaultDistBulkTransfer then //See bug in test/optimizations/bulkcomm/alberto/Block/rafatest2.chpl
```

Comment:
I can't tell what bug this comment means.  AFAICT, this test was never a future.

Reply:
(If not, then I think adjusting the path is sufficient for this PR)

Reply:
I this this comment should just be removed because:
1) Things have changed so much since 2012 (when this comment was added) that it is probably not accurate
2) I am unaware of a bug with that test